src/main/kotlin/com/msg/gauth/domain/oauth/services/OauthCodeService.kt

```diff
@@ -33,4 +35,15 @@ class OauthCodeService(
             code = code,
         )
     }
+
+    fun execute(): OauthCodeResponseDto{
+        val user = userUtil.fetchCurrentUser()
+        if(user.state != UserState.CREATED)
```

Comment:
조건식을 이런식으로 수정하는게 좋아보여요
```suggestion
        if(user.state == UserState.PENDING)
```

Reply:
<img width="934" alt="스크린샷 2023-03-10 오전 11 12 56" src="https://user-images.githubusercontent.com/80161826/224205771-ecafe3dd-7e2c-4585-8482-5aa9c1dd6921.png">